NYCPlanning/labs-zap-search
# 668: send hearing form to zap crm

### Changes:
- add new model attribute in `disposition` model `dcpIspublichearingrequired`
- update `waive-hearings-popup` to set `dcpIspublichearingrequired` on each disposition on that assignment to `No` if a user decides to opt out of hearings
- change `assignment.hearingsWaived` to check if `dcpIspublichearingrequired = 'No'`
- set `statuscode` disposition attribute to `Submitted` when the `submitRecommendations` action is run on `recommendations/add.hbs`
- remove `hearingsSubmitted` & `hearingsWaived` associated computed properties from `project` model, because those are now only associated with the `assignment` model

### Tests:
- update `add-test` for recommendations to check that `statuscode` is set to `Submitted` when `submitRecommendations` action is run
- update `to-review-project-card-test` to reflect new changes to waiving hearings
- remove `hearingsSubmitted` and `hearingsWaived` associated computed property tests from `project` model unit test and move into `assignment` model unit test

### To-Do:
- continue 2nd part of this issue in backend -- reference issue #668
